pages/common/mv.md

```diff
@@ -3,13 +3,17 @@
 > Move or rename files and directories.
 > More information: <https://www.gnu.org/software/coreutils/mv>.
 
-- Move a file to an arbitrary location:
+- Rename a file or directory when the target is not an existing directory:
 
 `mv {{source}} {{target}}`
```

Comment:
```suggestion
- Rename a file or directory when the target is not an existing file or directory:

`mv {{source}} {{target}}`
```

Reply:
No this is incorrect. Just keep it as before. Saying "if it is not an existing directory".

The thing is, if it is an existing file and you are renaming a file, then the existing file will be rewritten. If you are renaming a folder, on the other hand, then the `mv` command will come back with an error saying that `{{target}} is not a directory`